### PR TITLE
Relax MSP timeout

### PR DESCRIPTION
**Pull request by haslinghuis**
Perhaps 50ms is a tad low. So adjusting for 100ms. This is still an improvement as in the past we had a fixed 1000ms timeout.

Default in firmware:

`set serial_update_rate_hz = 100`

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by nerdCopter — APPROVED** [review]

**Review by ctzsnooze — APPROVED** [review]

**Pull request merged**
